esp/services/WsDeploy/WsDeployPlugin.cpp

```diff
@@ -44,7 +44,7 @@ extern "C"
 // Change the function names when we stick with dynamic loading.
 ESP_FACTORY IEspService * esp_service_factory(const char *name, const char* type, IPropertyTree *cfg, const char *process)
 {
-   if (strcmp(type, "WsDeploy")==0)
```

Comment:
is this related?  Something to note in the jira?

Reply:
Could have done without changing these calls.